platform/lib/platform/material.ex

```diff
@@ -110,14 +110,14 @@ defmodule Platform.Material do
     project_id = Keyword.get(opts, :restrict_to_project_id)
 
     filter_user =
-      if not is_nil(user) do
+      if user do
```

Comment:
Not quite equivalent. Per our conversation, can we do `is_nil` and then flip the if/else?

Reply:
changed to `is_nil` on the latest commit.